vgstation13.dme

```diff
@@ -2053,7 +2055,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_box.dm"
```

Comment:
One of these days someone's not going to catch this and the players will have to endure an actual round on test_box.